lib/ui/text.dart

```diff
@@ -106,184 +105,1306 @@ class FontWeight {
+  ///   // The Piazzolla font can be downloaded from Google Fonts (https://www.google.com/fonts).
+  ///   @override
+  ///   Widget build(BuildContext context) {
+  ///     return Text(
+  ///       'Call 311-555-2368 now!',
+  ///       style: TextStyle(
+  ///         fontFamily: 'Piazzolla',
+  ///         fontFeatures: <FontFeature>[
+  ///           FontFeature.tabularFigures(),
+  ///         ],
+  ///       ),
+  ///     );
+  ///   }
+  /// }
+  /// ```
+  ///
+  /// ![](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/font_feature_tnum.png)
+  /// {@end-tool}
+  ///
   /// See also:
   ///
   ///  * <https://docs.microsoft.com/en-us/typography/opentype/spec/features_pt#tnum>
   const FontFeature.tabularFigures() : feature = 'tnum', value = 1;
 
+  // zero
+  /// Use the slashed zero. (`zero`)
+  ///
+  /// Some fonts contain both a circular zero and a zero with a slash. This
+  /// enables the use of the latter form.
+  ///
+  /// This is overridden by [FontFeature.oldstyleFigures].
+  ///
+  /// {@tool sample --template=freeform}
+  ///
+  /// The Source Code Pro font supports the `zero` feature. It causes the
+  /// zero digit to be drawn with a slash rather than the default rendering,
+  /// which in this case has a dot through the zero rather than a slash.
+  ///
+  /// ```dart
+  /// class Demo extends StatelessWidget {
+  ///   // The Source Code Pro font can be downloaded from Google Fonts (https://www.google.com/fonts).
+  ///   @override
+  ///   Widget build(BuildContext context) {
+  ///     return Text(
+  ///       'One million is: 1,000,000.00',
+  ///       style: TextStyle(
+  ///         fontFamily: 'Source Code Pro',
+  ///         fontFeatures: <FontFeature>[
+  ///           FontFeature.slashedZero(),
+  ///         ],
+  ///       ),
+  ///     );
+  ///   }
+  /// }
+  /// ```
+  ///
+  /// ![](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/font_feature_zero.png)
+  /// {@end-tool}
+  ///
+  /// See also:
+  ///
+  ///  * <https://docs.microsoft.com/en-us/typography/opentype/spec/features_uz#zero>
+  const FontFeature.slashedZero() : feature = 'zero', value = 1;
+
+  // ------------------------------------------------------------------------
+  // End of feature tags list.
+
   /// The tag that identifies the effect of this feature.  Must consist of 4
   /// ASCII characters (typically lowercase letters).
   ///
-  /// See <https://docs.microsoft.com/en-us/typography/opentype/spec/featuretags>
+  /// These features are defined in a registry maintained by Microsoft:
+  /// <https://docs.microsoft.com/en-us/typography/opentype/spec/featuretags>
   final String feature;
 
   /// The value assigned to this feature.
   ///
-  /// Must be a positive integer.  Many features are Boolean values that accept
+  /// Must be a positive integer. Many features are Boolean values that accept
   /// values of either 0 (feature is disabled) or 1 (feature is enabled).
```

Comment:
nit: might want to mention here that other features use a bound range of acceptable values, and they're documented here for the ones we document or elsewhere for the ones we don't.

Reply:
added more color